### PR TITLE
Add translated files.

### PR DESCRIPTION
**Pull request by richardpringle**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,5 +17,5 @@ node_modules
 providers-private.json
 
 !intl/
```
Comment (0candy) [inline .gitignore]: Remove this line.

Comment (richardpringle) [inline .gitignore]: done

**Comment by richardpringle** [comment]
@slnode test please

**Comment by richardpringle** [comment]
@superkhau and cc/@candy

Is there a way that we can disable CI pass requirements just for this PR? We probably need to land these to make our globalization deadlines and in the meantime I can investigate the issue and make the fix.

**Comment by superkhau** [comment]
@0candy Thanks for overriding. @richardpringle Please make sure the fails are unrelated and link to an issue to re-enable at your discretion.

**Pull request merged**